.env

```diff
@@ -9,7 +9,7 @@ AUTH0_CLIENT_ID='Yjlt8LT5vXFJw1Z8m8eaB5aZO26uPyeD'
 PRISM_ENCRYPT_KEY='key'
 
 ENV='dev'
-DEBUG=True
+DEBUG=False
```

Comment:
? this is dev right?

Reply:
Oh, whoops! Left over from local testing - thanks.